File: MemoMaster/app/src/main/java/memomaster/lunastratos/com/memomaster/SplashActivity.java
package memomaster.lunastratos.com.memomaster;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.Nullable;

public class SplashActivity extends Activity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        Intent intent  = new Intent(getApplicationContext(), MainActivity.class);
        intent.putExtra("1", "1");
        startActivity(intent);
        finish();
    }
}

File: WhiteAlbum/app/src/main/java/memokotlin/lunastratos/com/whitealbum/SplashActivity.kt
package memokotlin.lunastratos.com.whitealbum

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity


class SplashActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val intent:Intent = Intent(this, MainActivity::class.java)
        intent.putExtra("start", "start")
        startActivity(intent)
        finish()
    }
}